File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/AgingTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class AgingTag extends Tag {
    public AgingTag(int count) {
        super(count, Constants.AGING_TAG, Constants.AGING_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/AgingTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class AgingTag(count: Int) : Tag(count, Constants.AGING_TAG, Constants.AGING_TAG_REMOTE, HashSet(listOf(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))